js/testapps/flow-simple-ai/src/index.ts

```diff
@@ -868,3 +888,18 @@ ai.defineFlow('geminiEnum', async (thing) => {
 
   return output;
 });
+
+ai.defineFlow('embedders-tester', async () => {
+  console.log(
+    await ai.embed({
+      content: 'hello world',
+      embedder: googleAI.embedder('text-embedding-004'),
```

Comment:
You could also take the content string and embedder name e.g. (vertexai/text-embedding-004) as input here, set these values as defaults, and make this more flexible while still not having to type anything every time you run it.

Reply:
same here, I'm testing `googleAI.embedder` function, not specific embedders.

Reply:
Sure, but this way you are only testing one possible input to the googleAI.embedder function. the other way, you can test various inputs. e.g. does it handle "googleai/text-embedding-004" or just "text-embedding-004"? What happens if you specify "vertexai/text-embedding-004"? What kind of errors do you get if you try with "text-embedding-banana"? Does it fail where you expect it to fail? What about special characters? Any other weird input you can think of to somehow screw it up?

Reply:
Same here. The ability to test more inputs is good.

Reply:
Same as above. This is not an e2e test. This is closer to sample code that we can refer user to showing them how to use these apis. Should be easy to read.